masterlist.yaml

```diff
@@ -8628,19 +8716,37 @@ plugins:
       - 'EpicGloryByMadmole.esp'
   - name: 'FactionOverhaulV25.esp'
     msg: [ *obsolete ]
+### Rules for Requiem and its patches ###
   - name: 'Requiem.esp'
+    global_priority: 75
```

Comment:
The rule of thumb is:
> Requiem should be the last mod in your load order to win potential conflicts with other mods. Only load other mods after Requiem, if you have a good reason to do so, like an explicit advice for Requiem from the author of the other mod.

It prevents all kinds of unintended side-effects, distorted gameplay and false bug reports.

Looking once more at other mods with a high `global_priority`, I think it could be reduced to 70 because all other mods with that priority require a Requiem patch anyway.